docs/introduction.md

```diff
@@ -1,19 +1,28 @@
 # Introduction
 
-[Wasmtime][github] is a [Bytecode Alliance][BA] project that is a standalone
-wasm-only optimizing runtime for [WebAssembly] and [WASI]. It runs WebAssembly
-code [outside of the Web], and can be used both as a command-line utility or as
-a library embedded in a larger application.
+[Wasmtime][github], a [Bytecode Alliance][BA] project, is used to run
+[Webassembly][Wasm] (Wasm) programs [without a web browser].  
+Typically, an application calls embedded Wasmtime to load and run Wasm
+[modules]. Or, the Wasmtime command line utility can run standalone Wasm
+programs.
 
-Wasmtime strives to be a highly configurable and embeddable runtime to run on
+[Webassembly][Wasm] (Wasm) is an assembly language for a [virtual
+stack machine][VSM] running inside an application.  High-level
+languages must be compiled to Wasm prior to loading. Upon loading,
+Wasmtime compiles Wasm to native machine code.
+
+[WebAssembly System Interface][WASI] (WASI) provides a library of
+essential functions, such as reading files, somewhat similar to libc.
```

Comment:
I don't think this is quite right: libc is an abstraction layer that can be implemented in terms of WASI, just as it can be implemented in terms of other system interfaces, such as POSIX.